src/periphery/UpgradeHelper.sol

```diff
@@ -9,17 +9,23 @@ import {IVault} from "../interfaces/IVault.sol";
 import {VaultTypeLib} from "../core/libs/VaultTypeLib.sol";
 import {VaultStatusLib} from "../core/libs/VaultStatusLib.sol";
 
+/// @title UpgradeHelper
+/// Changelog:
+///   2.0.0: Breaking change. Renamed the public variable `platform` to `PLATFORM`.
+/// @author Alien Deployer (https://github.com/a17)
+/// @author Jude (https://github.com/iammrjude)
 contract UpgradeHelper {
-    string public constant VERSION = "1.0.0";
+    string public constant VERSION = "2.0.0";
 
-    address public immutable platform;
```

Comment:
keep platform() to not have unneessacry change in interface, just ignore lint rule in contract